src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -1610,6 +1619,9 @@ void CEndlessUsbToolDlg::ErrorOccured(ErrorCause_t errorCause)
     case ErrorCause_t::ErrorCauseCanceled:
     case ErrorCause_t::ErrorCauseGeneric:
     case ErrorCause_t::ErrorCauseWriteFailed:
+    case ErrorCause_t::ErrorCauseNot64Bit:
+    case ErrorCause_t::ErrorCauseBitLocker:
+    case ErrorCause_t::ErrorCauseNotNTFS:
         buttonMsgId = MSG_328;
         suggestionMsgId = MSG_325;
```

Comment:
I have a branch which adds error messages for these cases, but doesn't actually wire them up – I'll rebase it after this is merged and wire them up here.